<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>测试报告</title>
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.0/dist/css/bootstrap.min.css"
          integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
    <!--    页面样式-->
    <style type="text/css">
        /*标题样式*/
        .title {
            width: auto;
            height: 60px;
            text-align: center;
            font: bolder 38px/60px "Microsoft YaHei UI";
        }

        /*汇总信息样式*/
        .summary {
            width: 90%;
            position: absolute;
            top: 120px;
            margin-left: 5%;

        }

        .text-left {
            font: bolder 20px/30px "Microsoft YaHei UI";
        }

        .left {
            width: 50%;
            float: left;

        }

        .right {
            width: 50%;
            float: right;

        }

        .desc {
            float: left;
            width: 100%;
        }

        .list-group-item span {
            font: normal 16px/38px "Microsoft YaHei UI";
            padding: 30px;

        }

        .list-group-item {
            position: relative;
            display: block;
            padding: .4rem 1.25rem;
            background-color: #fff;
            border: 1px solid rgba(0, 0, 0, .125);
        }

        /* 执行信息样式   */
        .test_info {
            width: 90%;
            position: absolute;
            top: 900px;
            margin-left: 5%;

            color: #28a745 !important;
        }

        .table td, th {
            border: solid 2px rgba(9, 122, 51, 0.11) !important;
            padding: 0;
            line-height: 40px;
            text-align: center;

        }


        select {
            border: 0;
            padding: 0;
            margin: 0;
            height: 2em;
            width: 8em;
            margin-left: 2em;
        }

        option {
            text-align: center;
            height: 36px;
            font: none 18px/36px "Microsoft YaHei UI";
            color: #28a745 !important;
        }

        .test_log {
            background: rgba(163, 171, 189, 0.15);
            width: 100%;
            height: 50px;
            border-top: none;
            border-bottom: none;
            display: none;
            text-align: left;
        }

        .test_log td {
            text-align: left;
            height: 30px;
            margin: 0;
            padding-left: 3em;
            font: none 18px/24px "Microsoft YaHei UI";
            color: #9e141a;
        }

        pre {
            margin: 0;
            white-space: pre-wrap;
            white-space: -moz-pre-wrap;
            white-space: -o-pre-wrap;
            word-wrap: break-word;
        }

        /*    测试图表显示*/
        .char {
            width: 90%;
            position: absolute;
            top: 450px;
            margin-left: 5%;
            color: #28a745 !important;
        }

    </style>

</head>
<body>

<!--报告标题-->
<div class="title text-success">
    <div class="shadow-lg p-3 mb-5 bg-white rounded">测试报告</div>
</div>

<!--汇总信息-->
<div class="summary">
    <p class="text-left text-success">测试结果汇总</p>
    <div class="left">
        <ul class="list-group">
            <li class="list-group-item">
                <button type="button" class="btn btn-success">测试人员</button>
                <span class="text-dark">孙忘</span>
            </li>
            <li class="list-group-item">
                <button type="button" class="btn btn-success">开始时间</button>
                <span class="text-dark">2020-09-06 16:36:49</span>
            </li>
            <li class="list-group-item">
                <button type="button" class="btn btn-success">执行时间</button>
                <span class="text-dark">28.62 S</span>
            </li>
            <li class="list-group-item">
                <button type="button" class="btn btn-success">用例总数</button>
                <span class="text-dark">126</span>
            </li>


        </ul>
    </div>
    <div class="right">
        <ul class="list-group">

            <li class="list-group-item">
                <button type="button" class="btn btn-success">成功用例</button>
                <span class="text-success">95</span>
            </li>
            <li class="list-group-item">
                <button type="button" class="btn btn-danger">失败用例</button>
                <span class="text-warning">31</span>
            </li>
            <li class="list-group-item">
                <button type="button" class="btn btn-warning">错误用例</button>
                <span class="text-danger">0</span>
            </li>
            <li class="list-group-item">
                <button type="button" class="btn btn-secondary">跳过用例</button>
                <span class="text-secondary">0</span>
            </li>
        </ul>
    </div>
    <div class="desc">
        <ul class="list-group">
            <li class="list-group-item">
                <button type="button" class="btn btn-success">描述信息</button>
                <span class="text-secondary">孙忘产生的测试报告</span>
            </li>
        </ul>
    </div>


</div>

<!--测试图表-->
<div class="char">
    <p class="text-left text-success">图表展示</p>
    <div id="char2" style="width: 49%;height: 400px;float: left"></div>
    <div id="char" style="width: 49%;height: 400px ;float: left"></div>
</div>

<!--详细信息-->
<div class="test_info">

    <p class="text-left text-success">详细信息</p>
    <div class="table_data">

        <table class="table">
            <thead class="bg-success text-light">
            <tr>
                <th scope="col" style="width: 5%;padding: 0">编号</th>
                <th scope="col" style="width: 20%;padding: 0">
                    <span>测试类</span>
                    <select id="testClass">
                        <option>所有</option>
                        
                            <option>TestLogin</option>
                        
                            <option>TestRecharge</option>
                        
                            <option>TestRegister</option>
                        
                            <option>TestWithdraw</option>
                        
                            <option>TestAdd</option>
                        
                            <option>TestAudit</option>
                        
                            <option>TestInvest</option>
                        
                            <option>TestLoans</option>
                        
                            <option>TestInfo</option>
                        
                            <option>TestUpdate</option>
                        
                    </select>

                </th>

                <th scope="col" style="width: 15%;padding: 0">测试方法</th>
                <th scope="col" style="width: 20%;padding: 0">用例描述</th>
                <th scope="col" style="width: 10%;padding: 0">执行时间</th>
                <th scope="col" style="width: 20%;padding: 0">

                    <span>执行结果</span>
                    <select id="testResult">
                        <option>所有</option>
                        <option class="text-success">成功</option>
                        <option class="text-danger">失败</option>
                        <option class="text-warning">错误</option>
                        <option class="text-info">跳过</option>
                    </select>

                </th>
                <th scope="col" style="width: 10%;padding: 0">详细信息</th>
            </tr>
            </thead>
            <tbody>
            
                <tr>
                    <td>1</td>
                    <td class="TestLogin">TestLogin</td>
                    <td>test_case_1</td>
                    <td>登录成功</td>
                    <td>0.184s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
test_case_1 (test_case.TestLogin)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>2</td>
                    <td class="TestLogin">TestLogin</td>
                    <td>test_case_2</td>
                    <td>登录失败_未注册的账号</td>
                    <td>0.227s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1001, 'msg': '账号信息错误'}
预期状态: {'code': 1001, 'msg': '账号信息错误'}
test_case_2 (test_case.TestLogin)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>3</td>
                    <td class="TestLogin">TestLogin</td>
                    <td>test_case_3</td>
                    <td>登录失败_错误的密码</td>
                    <td>0.152s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1001, 'msg': '账号信息错误'}
预期状态: {'code': 1001, 'msg': '账号信息错误'}
test_case_3 (test_case.TestLogin)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>4</td>
                    <td class="TestLogin">TestLogin</td>
                    <td>test_case_4</td>
                    <td>登录失败_多余的参数</td>
                    <td>0.153s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1001, 'msg': '账号信息错误'}
预期状态: {'code': 1001, 'msg': '账号信息错误'}
test_case_4 (test_case.TestLogin)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>5</td>
                    <td class="TestLogin">TestLogin</td>
                    <td>test_case_5</td>
                    <td>登录失败_账号为空</td>
                    <td>0.113s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1, 'msg': '手机号码为空'}
预期状态: {'code': 1, 'msg': '手机号码为空'}
test_case_5 (test_case.TestLogin)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>6</td>
                    <td class="TestLogin">TestLogin</td>
                    <td>test_case_6</td>
                    <td>登录失败_密码</td>
                    <td>0.158s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1, 'msg': '密码为空'}
预期状态: {'code': 1, 'msg': '密码为空'}
test_case_6 (test_case.TestLogin)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>7</td>
                    <td class="TestRecharge">TestRecharge</td>
                    <td>test_case_01</td>
                    <td>充值成功_0</td>
                    <td>0.383s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际充值: 0.0
预期充值: 0.0
实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
test_case_01 (test_case.TestRecharge)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>8</td>
                    <td class="TestRecharge">TestRecharge</td>
                    <td>test_case_02</td>
                    <td>充值成功_0.01</td>
                    <td>0.301s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际充值: 0.01
预期充值: 0.01
实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
test_case_02 (test_case.TestRecharge)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>9</td>
                    <td class="TestRecharge">TestRecharge</td>
                    <td>test_case_03</td>
                    <td>充值成功_499999.99</td>
                    <td>0.302s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际充值: 499999.99
预期充值: 499999.99
实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
test_case_03 (test_case.TestRecharge)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>10</td>
                    <td class="TestRecharge">TestRecharge</td>
                    <td>test_case_04</td>
                    <td>充值成功_500000</td>
                    <td>0.295s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际充值: 500000.0
预期充值: 500000.0
实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
test_case_04 (test_case.TestRecharge)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>11</td>
                    <td class="TestRecharge">TestRecharge</td>
                    <td>test_case_05</td>
                    <td>充值失败_小于0</td>
                    <td>0.205s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '余额必须大于0并且小于或者等于500000'}
预期状态: {'code': 2, 'msg': '金额必须大于0并且小于或者等于500000'}
test_case_05 (test_case.TestRecharge)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case.py", line 102, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case.py", line 98, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 43, in com_assertEqual
    self.assertEqual(respone["msg"], expect["msg"])  # msg
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 1233, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 693, in fail
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: '余额必须大于0并且小于或者等于500000' != '金额必须大于0并且小于或者等于500000'
- 余额必须大于0并且小于或者等于500000
? ^
+ 金额必须大于0并且小于或者等于500000
? ^

</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>12</td>
                    <td class="TestRecharge">TestRecharge</td>
                    <td>test_case_06</td>
                    <td>充值失败_大于50</td>
                    <td>0.163s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '余额必须大于0并且小于或者等于500000'}
预期状态: {'code': 2, 'msg': '金额必须大于0并且小于或者等于500000'}
test_case_06 (test_case.TestRecharge)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case.py", line 102, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case.py", line 98, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 43, in com_assertEqual
    self.assertEqual(respone["msg"], expect["msg"])  # msg
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 1233, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 693, in fail
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: '余额必须大于0并且小于或者等于500000' != '金额必须大于0并且小于或者等于500000'
- 余额必须大于0并且小于或者等于500000
? ^
+ 金额必须大于0并且小于或者等于500000
? ^

</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>13</td>
                    <td class="TestRecharge">TestRecharge</td>
                    <td>test_case_07</td>
                    <td>充值失败_精度小于小数点后两位</td>
                    <td>0.192s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '余额小数超过两位'}
预期状态: {'code': 2, 'msg': '金额小数超过两位'}
test_case_07 (test_case.TestRecharge)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case.py", line 102, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case.py", line 98, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 43, in com_assertEqual
    self.assertEqual(respone["msg"], expect["msg"])  # msg
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 1233, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 693, in fail
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: '余额小数超过两位' != '金额小数超过两位'
- 余额小数超过两位
? ^
+ 金额小数超过两位
? ^

</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>14</td>
                    <td class="TestRecharge">TestRecharge</td>
                    <td>test_case_08</td>
                    <td>充值失败_非登录账户id</td>
                    <td>0.228s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1007, 'msg': '无权限访问，请检查参数'}
预期状态: {'code': 1007, 'msg': '无权限访问，请检查参数'}
test_case_08 (test_case.TestRecharge)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>15</td>
                    <td class="TestRecharge">TestRecharge</td>
                    <td>test_case_09</td>
                    <td>充值失败_无登录账户id</td>
                    <td>0.119s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1, 'msg': '用户id为空'}
预期状态: {'code': 1, 'msg': '用户id为空'}
test_case_09 (test_case.TestRecharge)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>16</td>
                    <td class="TestRecharge">TestRecharge</td>
                    <td>test_case_10</td>
                    <td>充值失败_无充值金额</td>
                    <td>0.171s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '余额必须为数值型'}
预期状态: {'code': 2, 'msg': '必填参数为空'}
test_case_10 (test_case.TestRecharge)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case.py", line 102, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case.py", line 98, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 43, in com_assertEqual
    self.assertEqual(respone["msg"], expect["msg"])  # msg
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 1233, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 693, in fail
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: '余额必须为数值型' != '必填参数为空'
- 余额必须为数值型
+ 必填参数为空

</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>17</td>
                    <td class="TestRecharge">TestRecharge</td>
                    <td>test_case_11</td>
                    <td>充值失败_非数字金额</td>
                    <td>0.168s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '余额必须为数值型'}
预期状态: {'code': 2, 'msg': '金额必须为数值型'}
test_case_11 (test_case.TestRecharge)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case.py", line 102, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case.py", line 98, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 43, in com_assertEqual
    self.assertEqual(respone["msg"], expect["msg"])  # msg
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 1233, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 693, in fail
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: '余额必须为数值型' != '金额必须为数值型'
- 余额必须为数值型
? ^
+ 金额必须为数值型
? ^

</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>18</td>
                    <td class="TestRegister">TestRegister</td>
                    <td>test_case_01</td>
                    <td>注册成功_未注册的11位手机号_8位数字字母密码_权限空_昵称空</td>
                    <td>0.239s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
test_case_01 (test_case.TestRegister)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>19</td>
                    <td class="TestRegister">TestRegister</td>
                    <td>test_case_02</td>
                    <td>注册成功_未注册的11位手机号_16位数字字符密码_权限空_昵称空</td>
                    <td>0.221s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
test_case_02 (test_case.TestRegister)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>20</td>
                    <td class="TestRegister">TestRegister</td>
                    <td>test_case_03</td>
                    <td>注册失败_已存在的手机号</td>
                    <td>0.161s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '账号已存在'}
预期状态: {'code': 2, 'msg': '账号已存在'}
test_case_03 (test_case.TestRegister)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>21</td>
                    <td class="TestRegister">TestRegister</td>
                    <td>test_case_04</td>
                    <td>注册失败_已存在的手机号_不同密码</td>
                    <td>0.15s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '账号已存在'}
预期状态: {'code': 2, 'msg': '账号已存在'}
test_case_04 (test_case.TestRegister)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>22</td>
                    <td class="TestRegister">TestRegister</td>
                    <td>test_case_05</td>
                    <td>注册失败_账号为空</td>
                    <td>0.108s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1, 'msg': '手机号为空'}
预期状态: {'code': 1, 'msg': '手机号为空'}
test_case_05 (test_case.TestRegister)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>23</td>
                    <td class="TestRegister">TestRegister</td>
                    <td>test_case_06</td>
                    <td>注册失败_账号9位</td>
                    <td>0.121s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '无效的手机格式'}
预期状态: {'code': 2, 'msg': '无效的手机格式'}
test_case_06 (test_case.TestRegister)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>24</td>
                    <td class="TestRegister">TestRegister</td>
                    <td>test_case_07</td>
                    <td>注册失败_账号12位</td>
                    <td>0.118s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '无效的手机格式'}
预期状态: {'code': 2, 'msg': '无效的手机格式'}
test_case_07 (test_case.TestRegister)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>25</td>
                    <td class="TestRegister">TestRegister</td>
                    <td>test_case_08</td>
                    <td>注册失败_手机号带字母</td>
                    <td>0.116s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '无效的手机格式'}
预期状态: {'code': 2, 'msg': '无效的手机格式'}
test_case_08 (test_case.TestRegister)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>26</td>
                    <td class="TestRegister">TestRegister</td>
                    <td>test_case_09</td>
                    <td>注册失败_手机号带中文</td>
                    <td>0.117s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '无效的手机格式'}
预期状态: {'code': 2, 'msg': '无效的手机格式'}
test_case_09 (test_case.TestRegister)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>27</td>
                    <td class="TestRegister">TestRegister</td>
                    <td>test_case_10</td>
                    <td>注册失败_手机号带字符</td>
                    <td>0.11s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '无效的手机格式'}
预期状态: {'code': 2, 'msg': '无效的手机格式'}
test_case_10 (test_case.TestRegister)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>28</td>
                    <td class="TestRegister">TestRegister</td>
                    <td>test_case_11</td>
                    <td>注册失败_7位密码</td>
                    <td>0.212s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '密码格式为8到16位'}
预期状态: {'code': 2, 'msg': '密码格式为8到16位'}
test_case_11 (test_case.TestRegister)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>29</td>
                    <td class="TestRegister">TestRegister</td>
                    <td>test_case_12</td>
                    <td>注册失败_密码为空</td>
                    <td>0.206s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1, 'msg': '密码为空'}
预期状态: {'code': 1, 'msg': '密码为空'}
test_case_12 (test_case.TestRegister)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>30</td>
                    <td class="TestRegister">TestRegister</td>
                    <td>test_case_13</td>
                    <td>注册失败_17位密码</td>
                    <td>0.221s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '密码格式为8到16位'}
预期状态: {'code': 2, 'msg': '密码格式为8到16位'}
test_case_13 (test_case.TestRegister)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>31</td>
                    <td class="TestRegister">TestRegister</td>
                    <td>test_case_14</td>
                    <td>注册成功_权限0</td>
                    <td>0.229s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
test_case_14 (test_case.TestRegister)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>32</td>
                    <td class="TestRegister">TestRegister</td>
                    <td>test_case_15</td>
                    <td>注册成功_权限1</td>
                    <td>0.243s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
test_case_15 (test_case.TestRegister)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>33</td>
                    <td class="TestRegister">TestRegister</td>
                    <td>test_case_16</td>
                    <td>注册失败_权限2</td>
                    <td>0.221s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '不支持的用户类型'}
预期状态: {'code': 2, 'msg': '不支持的用户类型'}
test_case_16 (test_case.TestRegister)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>34</td>
                    <td class="TestRegister">TestRegister</td>
                    <td>test_case_17</td>
                    <td>注册失败_权限字母</td>
                    <td>0.231s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1004, 'msg': 'JSON parse error: Cannot deserialize value of type `java.lang.Integer` from String "alkj": not a valid Integer value; nested exception is com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `java.lang.Integer` from String "alkj": not a valid Integer value\n at [Source: (PushbackInputStream); line: 1, column: 59] (through reference chain: com.lemon.pojo.Member["type"])'}
预期状态: {'code': 2, 'msg': '内容无法解析'}
test_case_17 (test_case.TestRegister)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case.py", line 139, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case.py", line 135, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 39, in com_assertEqual
    self.assertEqual(respone["code"], expect["code"])  # 判断code
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: 1004 != 2
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>35</td>
                    <td class="TestRegister">TestRegister</td>
                    <td>test_case_18</td>
                    <td>注册失败_权限中文</td>
                    <td>0.28s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1004, 'msg': 'JSON parse error: Cannot deserialize value of type `java.lang.Integer` from String "管理员": not a valid Integer value; nested exception is com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `java.lang.Integer` from String "管理员": not a valid Integer value\n at [Source: (PushbackInputStream); line: 1, column: 59] (through reference chain: com.lemon.pojo.Member["type"])'}
预期状态: {'code': 2, 'msg': '内容无法解析'}
test_case_18 (test_case.TestRegister)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case.py", line 139, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case.py", line 135, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 39, in com_assertEqual
    self.assertEqual(respone["code"], expect["code"])  # 判断code
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: 1004 != 2
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>36</td>
                    <td class="TestRegister">TestRegister</td>
                    <td>test_case_19</td>
                    <td>注册成功_昵称1位</td>
                    <td>0.229s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
test_case_19 (test_case.TestRegister)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>37</td>
                    <td class="TestRegister">TestRegister</td>
                    <td>test_case_20</td>
                    <td>注册成功_昵称10位</td>
                    <td>0.227s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
test_case_20 (test_case.TestRegister)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>38</td>
                    <td class="TestRegister">TestRegister</td>
                    <td>test_case_21</td>
                    <td>注册失败_昵称11位</td>
                    <td>0.215s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '用户昵称长度超过10位'}
预期状态: {'code': 2, 'msg': '用户昵称长度超过10位'}
test_case_21 (test_case.TestRegister)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>39</td>
                    <td class="TestRegister">TestRegister</td>
                    <td>test_case_22</td>
                    <td>注册成功_昵称重复</td>
                    <td>0.421s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
test_case_22 (test_case.TestRegister)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>40</td>
                    <td class="TestWithdraw">TestWithdraw</td>
                    <td>test_case_01</td>
                    <td>提现成功_0.01</td>
                    <td>0.304s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际提现: 0.01
预期提现: 0.01
实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
test_case_01 (test_case.TestWithdraw)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>41</td>
                    <td class="TestWithdraw">TestWithdraw</td>
                    <td>test_case_02</td>
                    <td>提现成功_499999.99</td>
                    <td>0.296s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际提现: 499999.99
预期提现: 499999.99
实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
test_case_02 (test_case.TestWithdraw)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>42</td>
                    <td class="TestWithdraw">TestWithdraw</td>
                    <td>test_case_03</td>
                    <td>提现成功_500000</td>
                    <td>0.3s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际提现: 500000.0
预期提现: 500000.0
实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
test_case_03 (test_case.TestWithdraw)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>43</td>
                    <td class="TestWithdraw">TestWithdraw</td>
                    <td>test_case_04</td>
                    <td>提现失败_0</td>
                    <td>0.172s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 2, 'msg': '余额必须大于0并且小于或者等于5000000'}
0 != 2
test_case_04 (test_case.TestWithdraw)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case.py", line 50, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case.py", line 45, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 39, in com_assertEqual
    self.assertEqual(respone["code"], expect["code"])  # 判断code
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: 0 != 2
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>44</td>
                    <td class="TestWithdraw">TestWithdraw</td>
                    <td>test_case_05</td>
                    <td>提现失败_500000.01</td>
                    <td>0.149s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '余额必须大于0并且小于或者等于5000000'}
预期状态: {'code': 2, 'msg': '余额必须大于0并且小于或者等于5000000'}
test_case_05 (test_case.TestWithdraw)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>45</td>
                    <td class="TestWithdraw">TestWithdraw</td>
                    <td>test_case_06</td>
                    <td>提现失败_负数</td>
                    <td>0.145s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '余额必须大于0并且小于或者等于5000000'}
预期状态: {'code': 2, 'msg': '余额必须大于0并且小于或者等于5000000'}
test_case_06 (test_case.TestWithdraw)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>46</td>
                    <td class="TestWithdraw">TestWithdraw</td>
                    <td>test_case_07</td>
                    <td>提现失败_0.001</td>
                    <td>0.161s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '余额小数超过两位'}
预期状态: {'code': 2, 'msg': '金额小数超过两位'}
'余额小数超过两位' != '金额小数超过两位'
- 余额小数超过两位
? ^
+ 金额小数超过两位
? ^

test_case_07 (test_case.TestWithdraw)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case.py", line 50, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case.py", line 45, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 43, in com_assertEqual
    self.assertEqual(respone["msg"], expect["msg"])  # msg
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 1233, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 693, in fail
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: '余额小数超过两位' != '金额小数超过两位'
- 余额小数超过两位
? ^
+ 金额小数超过两位
? ^

</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>47</td>
                    <td class="TestWithdraw">TestWithdraw</td>
                    <td>test_case_08</td>
                    <td>提现失败_非登录id</td>
                    <td>0.242s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1002, 'msg': '余额不足'}
预期状态: {'code': 1007, 'msg': '无权限访问，请检查参数'}
1002 != 1007
test_case_08 (test_case.TestWithdraw)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case.py", line 50, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case.py", line 45, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 39, in com_assertEqual
    self.assertEqual(respone["code"], expect["code"])  # 判断code
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: 1002 != 1007
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>48</td>
                    <td class="TestWithdraw">TestWithdraw</td>
                    <td>test_case_09</td>
                    <td>提现失败_id为空</td>
                    <td>0.113s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1, 'msg': '会员id为空'}
预期状态: {'code': 1, 'msg': '会员id为空'}
test_case_09 (test_case.TestWithdraw)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>49</td>
                    <td class="TestWithdraw">TestWithdraw</td>
                    <td>test_case_10</td>
                    <td>提现失败_金额为空</td>
                    <td>0.152s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1, 'msg': '余额为空'}
预期状态: {'code': 1, 'msg': '金额为空'}
'余额为空' != '金额为空'
- 余额为空
? ^
+ 金额为空
? ^

test_case_10 (test_case.TestWithdraw)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case.py", line 50, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case.py", line 45, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 43, in com_assertEqual
    self.assertEqual(respone["msg"], expect["msg"])  # msg
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 1233, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 693, in fail
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: '余额为空' != '金额为空'
- 余额为空
? ^
+ 金额为空
? ^

</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>50</td>
                    <td class="TestWithdraw">TestWithdraw</td>
                    <td>test_case_11</td>
                    <td>提现失败_金额非整数</td>
                    <td>0.147s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '数值参数格式不正确'}
预期状态: {'code': 2, 'msg': '数值参数格式不正确'}
test_case_11 (test_case.TestWithdraw)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>51</td>
                    <td class="TestWithdraw">TestWithdraw</td>
                    <td>test_case_12</td>
                    <td>提现失败_id参数类型整数型</td>
                    <td>0.17s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 2, 'msg': '数值参数格式不正确'}
0 != 2
test_case_12 (test_case.TestWithdraw)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case.py", line 50, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case.py", line 45, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 39, in com_assertEqual
    self.assertEqual(respone["code"], expect["code"])  # 判断code
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: 0 != 2
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>52</td>
                    <td class="TestWithdraw">TestWithdraw</td>
                    <td>test_case_13</td>
                    <td>提现失败_金额大于余额</td>
                    <td>0.216s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1002, 'msg': '余额不足'}
预期状态: {'code': 1002, 'msg': '余额不足'}
test_case_13 (test_case.TestWithdraw)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>53</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_01</td>
                    <td>新增项目成功_1个月</td>
                    <td>0.237s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
实际项目名称: 孙忘_财务自由1
预期项目名称: 孙忘_财务自由1
test_case_01 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>54</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_02</td>
                    <td>新增项目成功_36个月</td>
                    <td>0.22s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
实际项目名称: 孙忘_财务自由2
预期项目名称: 孙忘_财务自由2
test_case_02 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>55</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_03</td>
                    <td>新增项目成功_10天</td>
                    <td>0.236s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
实际项目名称: 孙忘_财务自由3
预期项目名称: 孙忘_财务自由3
test_case_03 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>56</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_04</td>
                    <td>新增项目成功_45天</td>
                    <td>0.225s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
实际项目名称: 孙忘_财务自由4
预期项目名称: 孙忘_财务自由4
test_case_04 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>57</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_05</td>
                    <td>新增项目失败_不存在id</td>
                    <td>0.238s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '会员不存在'}
预期状态: {'code': 2, 'msg': '会员不存在'}
test_case_05 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>58</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_06</td>
                    <td>新增项目失败_非登录id</td>
                    <td>0.227s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1007, 'msg': '无权限访问，请检查参数'}
预期状态: {'code': 1007, 'msg': '无权限访问，请检查参数'}
test_case_06 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>59</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_07</td>
                    <td>新增项目失败_id类型错误</td>
                    <td>0.175s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 2, 'msg': '数值参数格式不正确'}
test_case_07 (test_case_add.TestAdd)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_add.py", line 44, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_add.py", line 35, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 39, in com_assertEqual
    self.assertEqual(respone["code"], expect["code"])  # 判断code
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: 0 != 2
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>60</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_08</td>
                    <td>新增项目失败_id空格</td>
                    <td>0.113s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1, 'msg': '用户id为空'}
预期状态: {'code': 1, 'msg': '用户id为空'}
test_case_08 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>61</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_09</td>
                    <td>新增项目失败_id为空</td>
                    <td>0.115s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1, 'msg': '用户id为空'}
预期状态: {'code': 1, 'msg': '用户id为空'}
test_case_09 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>62</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_10</td>
                    <td>新增项目失败_标题空格</td>
                    <td>0.149s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1, 'msg': '标题为空'}
预期状态: {'code': 1, 'msg': '标题为空'}
test_case_10 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>63</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_11</td>
                    <td>新增项目失败_标题为空</td>
                    <td>0.138s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1, 'msg': '标题为空'}
预期状态: {'code': 1, 'msg': '标题为空'}
test_case_11 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>64</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_12</td>
                    <td>新增项目失败_标题类型错误</td>
                    <td>0.157s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 2, 'msg': '数值参数格式不正确'}
test_case_12 (test_case_add.TestAdd)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_add.py", line 44, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_add.py", line 35, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 39, in com_assertEqual
    self.assertEqual(respone["code"], expect["code"])  # 判断code
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: 0 != 2
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>65</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_13</td>
                    <td>新增项目失败_借款为0</td>
                    <td>0.152s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '借款金额范围错误'}
预期状态: {'code': 2, 'msg': '借款金额范围错误'}
test_case_13 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>66</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_14</td>
                    <td>新增项目失败_借款为负数</td>
                    <td>0.143s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '借款金额范围错误'}
预期状态: {'code': 2, 'msg': '借款金额范围错误'}
test_case_14 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>67</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_15</td>
                    <td>新增项目失败_借款不为整数</td>
                    <td>0.153s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '借款金额范围错误'}
预期状态: {'code': 2, 'msg': '借款金额范围错误'}
test_case_15 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>68</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_16</td>
                    <td>新增项目失败_借款不为100的整数倍</td>
                    <td>0.158s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '借款金额范围错误'}
预期状态: {'code': 2, 'msg': '借款金额范围错误'}
test_case_16 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>69</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_17</td>
                    <td>新增项目失败_借款类型错误</td>
                    <td>0.168s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 2, 'msg': '数值参数格式不正确'}
test_case_17 (test_case_add.TestAdd)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_add.py", line 44, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_add.py", line 35, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 39, in com_assertEqual
    self.assertEqual(respone["code"], expect["code"])  # 判断code
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: 0 != 2
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>70</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_18</td>
                    <td>新增项目失败_借款为空</td>
                    <td>0.15s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1, 'msg': '借款金额为空'}
预期状态: {'code': 1, 'msg': '借款金额为空'}
test_case_18 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>71</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_19</td>
                    <td>新增项目失败_年化为0</td>
                    <td>0.161s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '借款利率范围错误'}
预期状态: {'code': 2, 'msg': '借款利率范围错误'}
test_case_19 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>72</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_20</td>
                    <td>新增项目失败_年化负数</td>
                    <td>0.157s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '借款利率范围错误'}
预期状态: {'code': 2, 'msg': '借款利率范围错误'}
test_case_20 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>73</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_21</td>
                    <td>新增项目失败_年化参数类型错误</td>
                    <td>0.161s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 2, 'msg': '数值参数格式不正确'}
test_case_21 (test_case_add.TestAdd)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_add.py", line 44, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_add.py", line 35, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 39, in com_assertEqual
    self.assertEqual(respone["code"], expect["code"])  # 判断code
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: 0 != 2
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>74</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_22</td>
                    <td>新增项目失败_年化参数为空</td>
                    <td>0.158s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1, 'msg': '利率为空'}
预期状态: {'code': 1, 'msg': '利率为空'}
test_case_22 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>75</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_23</td>
                    <td>新增项目失败_借款期限为0</td>
                    <td>0.155s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '借款期限单位为月的项目,期限范围为1到36个月(包含1个月和36个月)'}
预期状态: {'code': 2, 'msg': '借款期限单位为月的项目,期限范围为1到36个月(包含1个月和36个月)'}
test_case_23 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>76</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_24</td>
                    <td>新增项目失败_借款期限为小数</td>
                    <td>0.154s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '借款期限单位为月的项目,期限范围为1到36个月(包含1个月和36个月)'}
预期状态: {'code': 2, 'msg': '借款期限单位为月的项目,期限范围为1到36个月(包含1个月和36个月)'}
test_case_24 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>77</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_25</td>
                    <td>新增项目失败_借款类型错误</td>
                    <td>0.161s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 2, 'msg': '数值参数格式不正确'}
test_case_25 (test_case_add.TestAdd)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_add.py", line 44, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_add.py", line 35, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 39, in com_assertEqual
    self.assertEqual(respone["code"], expect["code"])  # 判断code
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: 0 != 2
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>78</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_26</td>
                    <td>新增项目失败_借款参数为空</td>
                    <td>0.15s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1, 'msg': '借款期限为空'}
预期状态: {'code': 1, 'msg': '借款期限为空'}
test_case_26 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>79</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_27</td>
                    <td>新增项目失败_借款期限为0天</td>
                    <td>0.164s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '借款期限单位为天的项目,期限范围为10到45天(包含10天和45天)'}
预期状态: {'code': 2, 'msg': '借款期限单位为天的项目,期限范围为10到45天(包含10天和45天)'}
test_case_27 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>80</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_28</td>
                    <td>新增项目失败_借款期限为9天</td>
                    <td>0.141s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '借款期限单位为天的项目,期限范围为10到45天(包含10天和45天)'}
预期状态: {'code': 2, 'msg': '借款期限单位为天的项目,期限范围为10到45天(包含10天和45天)'}
test_case_28 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>81</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_29</td>
                    <td>新增项目失败_借款期限为46天</td>
                    <td>0.124s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '借款期限单位为天的项目,期限范围为10到45天(包含10天和45天)'}
预期状态: {'code': 2, 'msg': '借款期限单位为天的项目,期限范围为10到45天(包含10天和45天)'}
test_case_29 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>82</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_30</td>
                    <td>新增项目失败_借款期限参数为空</td>
                    <td>0.153s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1, 'msg': '借款期限为空'}
预期状态: {'code': 1, 'msg': '借款期限为空'}
test_case_30 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>83</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_31</td>
                    <td>新增项目失败_借款期限单位为0</td>
                    <td>0.158s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '借款类型不存在'}
预期状态: {'code': 2, 'msg': '借款类型不存在'}
test_case_31 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>84</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_32</td>
                    <td>新增项目失败_借款期限单位为3</td>
                    <td>0.153s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '借款类型不存在'}
预期状态: {'code': 2, 'msg': '借款类型不存在'}
test_case_32 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>85</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_33</td>
                    <td>新增项目失败_借款期限单位类型错误</td>
                    <td>0.155s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '借款期限单位为月的项目,期限范围为1到36个月(包含1个月和36个月)'}
预期状态: {'code': 2, 'msg': '数值参数格式不正确'}
test_case_33 (test_case_add.TestAdd)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_add.py", line 44, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_add.py", line 35, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 43, in com_assertEqual
    self.assertEqual(respone["msg"], expect["msg"])  # msg
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 1233, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 693, in fail
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: '借款期限单位为月的项目,期限范围为1到36个月(包含1个月和36个月)' != '数值参数格式不正确'
- 借款期限单位为月的项目,期限范围为1到36个月(包含1个月和36个月)
+ 数值参数格式不正确

</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>86</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_34</td>
                    <td>新增项目失败_借款期限单位为空</td>
                    <td>0.153s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1, 'msg': '借款日期类型为空'}
预期状态: {'code': 1, 'msg': '借款日期类型为空'}
test_case_34 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>87</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_35</td>
                    <td>新增项目失败_竞标天数为0</td>
                    <td>0.156s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '竞标期限为1-10天'}
预期状态: {'code': 2, 'msg': '竞标期限为1-10天'}
test_case_35 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>88</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_36</td>
                    <td>新增项目失败_竞标天数为11</td>
                    <td>0.147s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '竞标期限为1-10天'}
预期状态: {'code': 2, 'msg': '竞标期限为1-10天'}
test_case_36 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>89</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_37</td>
                    <td>新增项目失败_竞标天数类型错误</td>
                    <td>0.171s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 2, 'msg': '数值参数格式不正确'}
test_case_37 (test_case_add.TestAdd)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_add.py", line 44, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_add.py", line 35, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 39, in com_assertEqual
    self.assertEqual(respone["code"], expect["code"])  # 判断code
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: 0 != 2
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>90</td>
                    <td class="TestAdd">TestAdd</td>
                    <td>test_case_38</td>
                    <td>新增项目失败_竞标天数参数为空</td>
                    <td>0.153s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1, 'msg': '竞标期限为空'}
预期状态: {'code': 1, 'msg': '竞标期限为空'}
test_case_38 (test_case_add.TestAdd)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>91</td>
                    <td class="TestAudit">TestAudit</td>
                    <td>test_case_1</td>
                    <td>审核成功_项目通过</td>
                    <td>0.367s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
实际项目状态: 2
预期项目状态: 2
test_case_1 (test_case_add.TestAudit)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>92</td>
                    <td class="TestAudit">TestAudit</td>
                    <td>test_case_2</td>
                    <td>审核成功_项目不通过</td>
                    <td>0.36s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
实际项目状态: 5
预期项目状态: 5
test_case_2 (test_case_add.TestAudit)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>93</td>
                    <td class="TestAudit">TestAudit</td>
                    <td>test_case_3</td>
                    <td>审核失败_项目已审核</td>
                    <td>0.272s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
实际状态: {'code': 2, 'msg': '项目不在审核中状态'}
预期状态: {'code': 2, 'msg': '项目不在审核中状态'}
test_case_3 (test_case_add.TestAudit)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>94</td>
                    <td class="TestAudit">TestAudit</td>
                    <td>test_case_4</td>
                    <td>审核失败_项目id类型错误</td>
                    <td>0.284s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 2, 'msg': '数值参数格式不正确'}
test_case_4 (test_case_add.TestAudit)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_add.py", line 115, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_add.py", line 101, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 39, in com_assertEqual
    self.assertEqual(respone["code"], expect["code"])  # 判断code
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: 0 != 2
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>95</td>
                    <td class="TestAudit">TestAudit</td>
                    <td>test_case_5</td>
                    <td>审核失败_项目id不存在</td>
                    <td>0.244s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
实际状态: {'code': 1, 'msg': '项目id为空'}
预期状态: {'code': 1, 'msg': '项目id为空'}
test_case_5 (test_case_add.TestAudit)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>96</td>
                    <td class="TestAudit">TestAudit</td>
                    <td>test_case_6</td>
                    <td>审核失败_审核状态类型错误</td>
                    <td>0.268s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
实际状态: {'code': 1, 'msg': 'approved_or_not只能为true或者false'}
预期状态: {'code': 1, 'msg': 'approved_or_not只能为true或者false'}
test_case_6 (test_case_add.TestAudit)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>97</td>
                    <td class="TestAudit">TestAudit</td>
                    <td>test_case_7</td>
                    <td>审核失败_审核状态类不存在</td>
                    <td>0.275s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
实际状态: {'code': 1, 'msg': 'approved_or_not为空'}
预期状态: {'code': 1, 'msg': 'approved_or_not为空'}
test_case_7 (test_case_add.TestAudit)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>98</td>
                    <td class="TestInvest">TestInvest</td>
                    <td>test_case_01</td>
                    <td>新增投资100成功</td>
                    <td>0.534s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
test_case_01 (test_case_invest.TestInvest)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>99</td>
                    <td class="TestInvest">TestInvest</td>
                    <td>test_case_02</td>
                    <td>新增投资失败_用户id参数类型错误</td>
                    <td>0.187s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 2, 'msg': '数值参数格式不正确'}
test_case_02 (test_case_invest.TestInvest)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_invest.py", line 59, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_invest.py", line 45, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 39, in com_assertEqual
    self.assertEqual(respone["code"], expect["code"])  # 判断code
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: 0 != 2
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>100</td>
                    <td class="TestInvest">TestInvest</td>
                    <td>test_case_03</td>
                    <td>新增投资失败_用户id参数为空</td>
                    <td>0.157s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1, 'msg': '会员id为空'}
预期状态: {'code': 1, 'msg': '项目id为空'}
test_case_03 (test_case_invest.TestInvest)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_invest.py", line 59, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_invest.py", line 45, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 43, in com_assertEqual
    self.assertEqual(respone["msg"], expect["msg"])  # msg
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 1233, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 693, in fail
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: '会员id为空' != '项目id为空'
- 会员id为空
+ 项目id为空

</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>101</td>
                    <td class="TestInvest">TestInvest</td>
                    <td>test_case_04</td>
                    <td>新增投资失败_非登录用户id</td>
                    <td>0.215s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1007, 'msg': '无权限访问，请检查参数'}
预期状态: {'code': 1007, 'msg': '无权限访问，请检查参数'}
test_case_04 (test_case_invest.TestInvest)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>102</td>
                    <td class="TestInvest">TestInvest</td>
                    <td>test_case_05</td>
                    <td>新增投资失败_非竞标项目</td>
                    <td>0.215s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '该项目不在竞标中状态'}
预期状态: {'code': 2, 'msg': '该项目不在竞标中状态'}
test_case_05 (test_case_invest.TestInvest)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>103</td>
                    <td class="TestInvest">TestInvest</td>
                    <td>test_case_06</td>
                    <td>新增投资失败_竞标项目参数为空</td>
                    <td>0.162s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1, 'msg': '项目id为空'}
预期状态: {'code': 2, 'msg': '项目id为空'}
test_case_06 (test_case_invest.TestInvest)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_invest.py", line 59, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_invest.py", line 45, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 39, in com_assertEqual
    self.assertEqual(respone["code"], expect["code"])  # 判断code
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: 1 != 2
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>104</td>
                    <td class="TestInvest">TestInvest</td>
                    <td>test_case_07</td>
                    <td>新增投资失败_投资金额非100整数</td>
                    <td>0.152s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '充值金额必须大于100，并且可以被100整除'}
预期状态: {'code': 2, 'msg': '充值金额必须大于100，并且可以被100整除'}
test_case_07 (test_case_invest.TestInvest)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>105</td>
                    <td class="TestInvest">TestInvest</td>
                    <td>test_case_08</td>
                    <td>新增投资失败_投资金额超出最大金额</td>
                    <td>0.153s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '该标可投金额不足,可投金额：9.99998E7'}
预期状态: {'code': 2, 'msg': '该标可投金额不足'}
test_case_08 (test_case_invest.TestInvest)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_invest.py", line 59, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_invest.py", line 45, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 41, in com_assertEqual
    self.assertEqual(respone["msg"], expect["msg"].split(",")[0])  # msg
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 1233, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 693, in fail
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: '该标可投金额不足,可投金额：9.99998E7' != '该标可投金额不足'
- 该标可投金额不足,可投金额：9.99998E7
+ 该标可投金额不足

</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>106</td>
                    <td class="TestInvest">TestInvest</td>
                    <td>test_case_09</td>
                    <td>新增投资失败_投资金额负数</td>
                    <td>0.154s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '充值金额必须大于100，并且可以被100整除'}
预期状态: {'code': 2, 'msg': '充值金额必须大于100，并且可以被100整除'}
test_case_09 (test_case_invest.TestInvest)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>107</td>
                    <td class="TestInvest">TestInvest</td>
                    <td>test_case_10</td>
                    <td>新增投资失败_投资金额为0</td>
                    <td>0.151s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '充值金额必须大于100，并且可以被100整除'}
预期状态: {'code': 2, 'msg': '充值金额必须大于100，并且可以被100整除'}
test_case_10 (test_case_invest.TestInvest)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>108</td>
                    <td class="TestInvest">TestInvest</td>
                    <td>test_case_11</td>
                    <td>新增投资失败_投资金额参数错误</td>
                    <td>0.167s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 2, 'msg': '数值参数格式不正确'}
test_case_11 (test_case_invest.TestInvest)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_invest.py", line 59, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_invest.py", line 45, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 39, in com_assertEqual
    self.assertEqual(respone["code"], expect["code"])  # 判断code
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: 0 != 2
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>109</td>
                    <td class="TestInvest">TestInvest</td>
                    <td>test_case_12</td>
                    <td>新增投资失败_投资金额参数为空</td>
                    <td>0.166s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1, 'msg': '余额为空'}
预期状态: {'code': 2, 'msg': '金额为空'}
test_case_12 (test_case_invest.TestInvest)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_invest.py", line 59, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_invest.py", line 45, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 39, in com_assertEqual
    self.assertEqual(respone["code"], expect["code"])  # 判断code
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: 1 != 2
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>110</td>
                    <td class="TestInvest">TestInvest</td>
                    <td>test_case_13</td>
                    <td>新增投资最大值成功</td>
                    <td>0.509s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
test_case_13 (test_case_invest.TestInvest)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>111</td>
                    <td class="TestLoans">TestLoans</td>
                    <td>test_case_1</td>
                    <td>新增获取项目列表成功</td>
                    <td>0.137s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
test_case_1 (test_case_invest.TestLoans)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>112</td>
                    <td class="TestLoans">TestLoans</td>
                    <td>test_case_2</td>
                    <td>新增获取项目列表成功</td>
                    <td>0.12s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
test_case_2 (test_case_invest.TestLoans)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>113</td>
                    <td class="TestLoans">TestLoans</td>
                    <td>test_case_3</td>
                    <td>新增获取项目列表成功</td>
                    <td>0.123s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
test_case_3 (test_case_invest.TestLoans)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>114</td>
                    <td class="TestLoans">TestLoans</td>
                    <td>test_case_4</td>
                    <td>新增获取项目列表失败_索引参数类型错误</td>
                    <td>0.136s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 2, 'msg': '数值参数格式不正确'}
test_case_4 (test_case_invest.TestLoans)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_invest.py", line 100, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_invest.py", line 96, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 39, in com_assertEqual
    self.assertEqual(respone["code"], expect["code"])  # 判断code
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: 0 != 2
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>115</td>
                    <td class="TestLoans">TestLoans</td>
                    <td>test_case_5</td>
                    <td>新增获取项目列表失败_每页大小类型错误</td>
                    <td>0.126s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 2, 'msg': '数值参数格式不正确'}
test_case_5 (test_case_invest.TestLoans)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_invest.py", line 100, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_invest.py", line 96, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 39, in com_assertEqual
    self.assertEqual(respone["code"], expect["code"])  # 判断code
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: 0 != 2
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>116</td>
                    <td class="TestInfo">TestInfo</td>
                    <td>test_case_1</td>
                    <td>获取信息成功</td>
                    <td>0.139s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
test_case_1 (test_case_update.TestInfo)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>117</td>
                    <td class="TestUpdate">TestUpdate</td>
                    <td>test_case_01</td>
                    <td>更新昵称成功_1位</td>
                    <td>0.268s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际昵称: 孙
预期昵称: 孙
实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 0, 'msg': 'OK'}
test_case_01 (test_case_update.TestUpdate)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>118</td>
                    <td class="TestUpdate">TestUpdate</td>
                    <td>test_case_02</td>
                    <td>更新昵称成功_10位</td>
                    <td>0.267s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际昵称: 孙
预期昵称: 孙孙孙孙孙孙孙孙孙孙
test_case_02 (test_case_update.TestUpdate)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_update.py", line 44, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_update.py", line 39, in test_case
    self.assertEqual(res["reg_name"], self.params["reg_name"])  # 校验更新后的昵称与预期
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 1233, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 693, in fail
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: '孙' != '孙孙孙孙孙孙孙孙孙孙'
- 孙
+ 孙孙孙孙孙孙孙孙孙孙

</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>119</td>
                    <td class="TestUpdate">TestUpdate</td>
                    <td>test_case_03</td>
                    <td>更新昵称失败_id 非整数型</td>
                    <td>0.173s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 2, 'msg': '数值参数格式不正确'}
test_case_03 (test_case_update.TestUpdate)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_update.py", line 44, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_update.py", line 40, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 39, in com_assertEqual
    self.assertEqual(respone["code"], expect["code"])  # 判断code
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: 0 != 2
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>120</td>
                    <td class="TestUpdate">TestUpdate</td>
                    <td>test_case_04</td>
                    <td>更新昵称失败_id 非整数型</td>
                    <td>0.113s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '数值参数格式错误'}
预期状态: {'code': 2, 'msg': '数值参数格式错误'}
test_case_04 (test_case_update.TestUpdate)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>121</td>
                    <td class="TestUpdate">TestUpdate</td>
                    <td>test_case_05</td>
                    <td>更新昵称失败_id 非登录</td>
                    <td>0.225s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1007, 'msg': '无权限访问，请检查参数'}
预期状态: {'code': 1007, 'msg': '无权限访问，请检查参数'}
test_case_05 (test_case_update.TestUpdate)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>122</td>
                    <td class="TestUpdate">TestUpdate</td>
                    <td>test_case_06</td>
                    <td>更新昵称失败_id 为空</td>
                    <td>0.113s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1, 'msg': '用户id为空'}
预期状态: {'code': 1, 'msg': '用户id为空'}
test_case_06 (test_case_update.TestUpdate)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>123</td>
                    <td class="TestUpdate">TestUpdate</td>
                    <td>test_case_07</td>
                    <td>更新昵称失败_昵称11位</td>
                    <td>0.157s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 2, 'msg': '用户昵称长度超过10位'}
预期状态: {'code': 2, 'msg': '用户昵称长度超过10位'}
test_case_07 (test_case_update.TestUpdate)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>124</td>
                    <td class="TestUpdate">TestUpdate</td>
                    <td>test_case_08</td>
                    <td>更新昵称失败_昵称为空</td>
                    <td>0.152s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1, 'msg': '用户名为空'}
预期状态: {'code': 1, 'msg': '用户名为空'}
test_case_08 (test_case_update.TestUpdate)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>125</td>
                    <td class="TestUpdate">TestUpdate</td>
                    <td>test_case_09</td>
                    <td>更新昵称失败_昵称为空</td>
                    <td>0.144s</td>
                    
                        <td class="text-success">成功</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 1, 'msg': '用户名不能为空字符串，也不能全部是空格'}
预期状态: {'code': 1, 'msg': '用户名不能为空字符串，也不能全部是空格'}
test_case_09 (test_case_update.TestUpdate)执行——>【通过】
</pre>
                        
                        
                    </td>
                </tr>
            
                <tr>
                    <td>126</td>
                    <td class="TestUpdate">TestUpdate</td>
                    <td>test_case_10</td>
                    <td>更新昵称失败_昵称非字符串</td>
                    <td>0.184s</td>
                    
                        <td class="text-danger">失败</td>
                    
                    <td>
                        <button type="button" class="btn btn-success btn_info">查看详情</button>
                    </td>
                </tr>

                <tr class="test_log">

                    <td colspan="7">
                        

                            <pre>实际状态: {'code': 0, 'msg': 'OK'}
预期状态: {'code': 2, 'msg': '数值参数格式不正确'}
test_case_10 (test_case_update.TestUpdate)执行——>【失败】
</pre>
                        

                            <pre>Traceback (most recent call last):
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 59, in testPartExecutor
    yield
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 628, in run
    testMethod()
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_update.py", line 44, in test_case
    raise e
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\testcase\test_case_update.py", line 40, in test_case
    com_assertEqual(self, respone, eval(kwargs["expect"]))
</pre>
                        

                            <pre>  File "c:\工具资料\GitHub\python_api_autotest\com_func\com_request_func.py", line 39, in com_assertEqual
    self.assertEqual(respone["code"], expect["code"])  # 判断code
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 852, in assertEqual
    assertion_func(first, second, msg=msg)
</pre>
                        

                            <pre>  File "C:\Users\64121\AppData\Local\Programs\Python\Python37\lib\unittest\case.py", line 845, in _baseAssertEqual
    raise self.failureException(msg)
</pre>
                        

                            <pre>AssertionError: 0 != 2
</pre>
                        
                        
                    </td>
                </tr>
            

            </tbody>
        </table>
    </div>

    <div style="height: 200px"></div>
</div>

<script src="https://cdn.jsdelivr.net/npm/jquery@3.5.1/dist/jquery.slim.min.js"
        integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
        crossorigin="anonymous"></script>
<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/echarts/dist/echarts.min.js"></script>

<script>
    var tbodyTr = $('tbody tr');
    var testResult = $("#testResult");
    var testClass = $("#testClass");
    <!--    用例执行详细信息显示切换-->
    $(".btn_info").click(function () {
        $(this).parent().parent().next().toggle();

    });
    // 当选择用例类之后触发
    testClass.change(function () {
        var cls = $(this).val();
        var res = testResult.val();
        elementDisplay(cls, res);
        sort()
    });
    testResult.change(function () {
        var res = $(this).val();
        var cls = testClass.val();
        elementDisplay(cls, res);
        sort()
    });

    function elementDisplay(cls, res) {
        // 用例数据的显示
        if (cls === "所有") {
            if (res === "所有") {
                tbodyTr.has('button').show();
            } else if (res === '成功') {
                tbodyTr.hide();
                tbodyTr.has('button').has('.text-success').show()

            } else if (res === '失败') {
                tbodyTr.hide();
                tbodyTr.has('button').has('.text-danger').show()

            } else if (res === '错误') {
                tbodyTr.hide();
                tbodyTr.has('button').has('.text-warning').show()

            } else if (res === '跳过') {
                tbodyTr.hide();
                tbodyTr.has('button').has('.text-info').show()
            }
        } else {
            if (res === "所有") {
                tbodyTr.hide();
                tbodyTr.has('button').has('.' + cls + '').show()
            } else if (res === '成功') {
                tbodyTr.hide();
                tbodyTr.has('button').has('.' + cls + '').has('.text-success').show()
            } else if (res === '失败') {
                tbodyTr.hide();
                tbodyTr.has('button').has('.' + cls + '').has('.text-danger').show()
            } else if (res === '错误') {
                tbodyTr.hide();
                tbodyTr.has('button').has('.' + cls + '').has('.text-warning').show()
            } else if (res === '跳过') {
                tbodyTr.hide();
                tbodyTr.has('button').has('.' + cls + '').has('.text-info').show()
            }
        }
    }

    function sort() {
        //重新排列显示序号
        // 选择所有可以见的tr
        var visibleTr = tbodyTr.filter(":visible");

        visibleTr.each(function (index, element) {
            element.firstElementChild.innerHTML = index + 1;

        })

    }


</script>

<script type="text/javascript">
    // 基于准备好的dom，初始化echarts实例
    var myChart = echarts.init(document.getElementById('char'));
    var myChart2 = echarts.init(document.getElementById('char2'));
    // 指定图表的配置项和数据
    option = {
        color: ['#00a10a', '#ddb518', 'rgba(204,46,41,0.73)', '#85898c'],
        tooltip: {
            trigger: 'item',
            formatter: '{a} <br/>{b}: {c} ({d}%)'
        },
        legend: {
            orient: 'vertical',
            left: 10,
            data: ['通过', '失败', '错误', '跳过']
        },
        series: [
            {
                name: '测试结果',
                type: 'pie',
                radius: ['50%', '70%'],
                avoidLabelOverlap: false,
                label: {
                    show: false,
                    position: 'center'
                },
                emphasis: {
                    label: {
                        show: true,
                        fontSize: '30',
                        fontWeight: 'bold'
                    }
                },
                labelLine: {
                    show: false
                },
                data: [
                    {value: 95, name: '通过'},
                    {value: 31, name: '失败'},
                    {value: 0, name: '错误'},
                    {value: 0, name: '跳过'}
                ]
            }
        ]
    };
    option2 = {
        tooltip: {
            formatter: '{a} <br/>{b} : {c}%'
        },
        toolbox: {
            feature: {
                restore: {},
                saveAsImage: {}
            }
        },
        series: [
            {
                name: '测试结果',
                type: 'gauge',
                detail: {formatter: '75.40%'},
                data: [{value: '75.40', name: '用例通过率'}],
                axisLine: {
                    lineStyle: {
                        color: [
                            [0.2, '#c20000'],
                            [0.8, '#ddb518'],
                            [1, '#00a10a']]
                    }
                }
            }
        ]
    };


    myChart2.setOption(option2);
    // 使用刚指定的配置项和数据显示图表。
    myChart.setOption(option);
</script>


</body>
</html>